etc/skeleton/driver/normalizer/parser.go

```diff
@@ -5,12 +5,29 @@ import (
 	"github.com/bblfsh/sdk/protocol/native"
 )
 
+// ToNoder specifies the driver options. Driver programmers should fill it
+var ToNoder = &native.ObjectToNoder{}
+
 // ParserBuilder creates a parser that transform source code files into *uast.Node.
-func ParserBuilder(opts driver.ParserOptions) (driver.Parser, error) {
-	toNoder := &native.ObjectToNoder{}
-	parser, err := native.ExecParser(toNoder, opts.NativeBin)
+func ParserBuilder(opts driver.ParserOptions) (parser driver.Parser, err error) {
+	psr, err := native.ExecParser(ToNoder, opts.NativeBin)
```

Comment:
There's no need for a different (and harder to understand) `psr` name, is it? You can just reassign `parser` if needed.

Reply:
I wanted to differentiate the normal parser from the transformation parsers that take the first as one of their fields (2 and 3 cases in the switch), but maybe using a single parser var is clearer.